src/components/ProductCard/index.js

```diff
@@ -1,8 +1,60 @@
 import React from 'react';
-import './styles.css';
+import PropTypes from 'prop-types';
+import Timer from '../Timer';
+import GroupOccupancy from '../GroupOccupancy';
+import './styles.scss';
 
-const ProductCard = () => {
-    
+const ProductCard = ({
+    productPhotoUrl,
+    listPrice,
+    discountedPrice,
+    discountPercentage,
+    expirationDate,
+    description,
+    colors,
+    groupCapacity,
+    subscriptorsNumber,
+    occupancyPercentage,
+    extended
+}) => {
+    return (
+        <div>
+            <img src={productPhotoUrl} alt="Imagen" />
```

Comment:
I would put `alt='Imagen del producto'`. Otherwise a screen reader just reads "Imagen", I think that is ambiguous